<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dygma Raise Assembly Tests</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="logoLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/images/Dygma_logo_color.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="light">
         <attribute name="title">
          <string>Light control</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="0">
             <widget class="QPushButton" name="light_white">
              <property name="text">
               <string>White</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="light_red">
              <property name="text">
               <string>Red</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="light_blue">
              <property name="text">
               <string>Blue</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="light_green">
              <property name="text">
               <string>Green</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="light_off">
              <property name="text">
               <string>Off</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="magnet">
         <attribute name="title">
          <string>Magnet Calibrate</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QPushButton" name="magnet_joined">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Keyboard is Joined</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="magnet_split">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Keyboard is Split</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>individual LED</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QPushButton" name="led_next">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Next</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="led_prev">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Prev</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLCDNumber" name="led_num"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="log_messages">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>489</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDygma_Raise_Assembly_Tests">
    <property name="title">
     <string>Dygma Raise Assembly Tests</string>
    </property>
   </widget>
   <addaction name="menuDygma_Raise_Assembly_Tests"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
